### PR TITLE
Tidy with ESLint

### PR DESCRIPTION
**Pull request by christianbundy**
This resolves a few small errors found with the default recommended
ESLint config, and adds a comment explaining a piece of code that I
found confusing (and appears to lack any kind of tests or
documentation).

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ test('initialize, but append before peerClock loads', function (t) {
   state = events.connect(state, {id: 'alice', client: false})
   state = events.append(state, {author: 'bob', sequence: 3, content: {}})
 
-  state = events.peerClock(state, {id: 'alice', value: {}})
+  events.peerClock(state, {id: 'alice', value: {}})
```
Comment (arj03) [inline test/index.js:130]: I liked this better the way it was before. peerClock returns state.

**Comment by christianbundy** [comment]
Thanks for the merge!

**Comment by arj03** [comment]
Thank you both for cleaning up the code :heart:  Looks much nicer. I pushed a small change for my commit after the merge.

**Comment by christianbundy** [comment]
Works for me!

**Pull request merged**